src/cmd/cmd_c.ml

```diff
@@ -33,6 +33,59 @@ let find location file : Fpath.t Result.t =
   in
   loop l
 
+let eacsl_instrument eacsl debug ~includes (files : Fpath.t list) :
+  Fpath.t list Result.t =
+  if eacsl then
+    let flags1 =
+      let includes =
+        String.concat " "
+          (List.map (fun libpath -> "-I" ^ Fpath.to_string libpath) includes)
+      in
+      let framac_verbosity_level = if debug then "2" else "0" in
+      Cmd.(
+        of_list
+          [ "-e-acsl"
+          ; "-no-frama-c-stdlib"
+          ; "-kernel-warn-key"
+          ; "CERT:MSC:38=inactive"
+          ; "-verbose"
+          ; framac_verbosity_level
+          ; String.concat "" [ "-cpp-extra-args=\""; includes; "\"" ]
```

Comment:
You can rewrite this like this to avoid quoting:

```ocaml
; String.concat "" [ {|-cpp-extra-args="|}; includes; {|"|} ]
```

Reply:
Thanks!